build/Targets/Settings.props

```diff
@@ -22,6 +22,7 @@
     <RoslynPortableTargetFrameworks>net46;netcoreapp2.0</RoslynPortableTargetFrameworks>
  
     <Features>strict,IOperation</Features>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
```

Comment:
➡️ Looks like this is [not necessary](https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md#msbuild):

> If you encounter a problem using reference assemblies, you can set the boolean property CompileUsingReferenceAssemblies to false to avoid using ref assemblies even if the projects you reference produce them. This is unset by default and only ever checked against false. It is only there to provide an emergency escape hatch; a customer who hits a bug can set it to false and avoid the new codepaths.